src/main/java/dev/aura/bungeechat/listener/BungeeChatEventsListener.java

```diff
@@ -1,11 +1,12 @@
 package dev.aura.bungeechat.listener;
 
+import com.google.common.collect.MapMaker;
 import dev.aura.bungeechat.event.BungeeChatJoinEvent;
 import dev.aura.bungeechat.event.BungeeChatLeaveEvent;
 import dev.aura.bungeechat.event.BungeeChatServerSwitchEvent;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.UUID;
+
+import java.util.*;
```

Comment:
Please run `./gradlew spotlessApply` this will only explicitly import stuff that’s needed.  
No wildcard imports!

Reply:
I've run it and updated this PR.